### ISSUE TITLE
All legends coming up black

### ISSUE THREAD

**Issue by markskayff** [flot/flot#1708]
I have all legends coming up with a black background and there's seem to be no control whatsoever on the legend background color and opacity. The source code shows no control over it, neither it's showing the defaults on this values.

![legends-coming-black](https://user-images.githubusercontent.com/22687247/66337413-751e0f00-e915-11e9-8390-59fc19168f45.png)

**Comment by markskayff** [comment]
Commenting my own issue.
What I found is that I couldn't control the SVG rect shape for the legend background with the CSS "background" property. I had to do it with the SVG "fill" property. 

Not sure if this is a new spec from most recent browser engines, or like a bug or what. But had to patch the script to use the "fill" property to control this.

**Comment by seba-salavilla** [comment]
I solved adding style

> .legendLayer .background {
    fill: rgba(255, 255, 255, 0.85);
    stroke: rgba(0, 0, 0, 0.85);
    stroke-width: 1;
}

**Comment by roben** [comment]
I can confirm this issue using Chrome 78. Thanks for the workaround @seba-salavilla!

**Comment by simonmurdock** [comment]
Also having this problem, Seems very odd.

**Comment by HaraldKi** [comment]
Same for firefox 71.0 on Linux. To reproduce, take out the all .legend and .legendLayer definitions from the examples.css and add a legend to the plot (tried with Basic Usage).

**Comment by ktecho** [comment]
Yeah, it's happening to me too. Black background and black labels, so nothing can be seen.
